raiden-network/light-client
# dApp: Nicer looking info icon on empty withdrawal screen

Fixes #1613 

**Short description**
Replaces the material design info icon with a custom SVG.
<img width="528" alt="Screenshot 2020-06-15 at 17 06 26" src="https://user-images.githubusercontent.com/43838780/84673908-961a1380-af2a-11ea-8078-4995a261c8a6.png">


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Open the Account screen and click the Withdrawal option.
2. The icon will appear if there are no tokens available for withdrawal.
